.github/workflows/ci.yaml

```diff
@@ -35,6 +35,10 @@ jobs:
       - name: logs on fail
         if: ${{ failure() }}
         run: lando logs -s core
+      - name: Codecov
+        uses: codecov/codecov-action@v1.0.5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
How comes that this is a change?

Reply:
Missed that in #152 and now it was bringing the % down